scraping/batch_notify.py

```
@@ -0,0 +1,44 @@
+    if missing <= threshold:
+        logger.info(
+            f"batch notify: missing={missing} under threshold={threshold}"
+        )
+        return
+
+    text = f"[velog-dashboard-v2] 오늘 통계 누락 포스트 {missing}건이 임계({threshold}) 초과"
+    try:
+        redis_client = None
+        try:
+            redis_client = get_redis_client()
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)